setupTests.js

```diff
@@ -5,16 +5,50 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+let hathorMemoryStorage = {};
+// Creating memory storage to be used in the place of localStorage
+const storageFactory = {
```

Comment:
Isn't it better to use a class instead of an object? We know they are almost the same, but the implementation would be easier to read and understand.

Reply:
Not sure I follow. Why is it easier to read and understand?

Reply:
You had to create a global variable and set the attributes there. I think the encapsulation form is simpler and less error prone (even though this code is very simple).

Reply:
Done